lib/spack/spack/solver/concretize.lp

```diff
@@ -1088,156 +1121,168 @@ opt_criterion(100, "number of packages to build (vs. reuse)").
 #defined optimize_for_reuse/0.
 
 % Minimize the number of deprecated versions being used
-opt_criterion(15, "deprecated versions used").
-#minimize{ 0@215: #true }.
-#minimize{ 0@15: #true }.
+opt_criterion(75, "deprecated versions used").
```

Comment:
Can you explain why all this renumbering was necessary?

Reply:
At one point, I had prioritized this differently and found myself in a scenario akin to `ArrayList.insert(0, item)` except I was the computer. I created "space" between these priority rankings so that other users can insert new constraints wherever they want  without having to shift everything by 1 (at some point this isn't possible, but I figured I'd help out the next 2-3 people).